src/guide/modules.md

```diff
@@ -100,32 +100,32 @@ from "list" include List as Stdlist
 Stdlist.length([1, 2, 3]) // 3
 ```
 
-### Including All Values
+### Bring Values from a Module into Scope
 
-If we want to bring all of the values from another module into scope in our program, we can use an asterisk.
+Individual items in a module can be brought into scope by using the `use` keyword, placing the desired items in curly braces. Similar to with the `provide { ... }` syntax, we need to prefix types with `type` and modules with `module`
 
 ```grain
 module Main
 
-from "list" include List
+from "array" include Array
 
-use List.*
+use Array.{ length, module Immutable }
 
-length([1, 2, 3]) // 3
-reverse([1, 2, 3]) // [3, 2, 1]
+length([> 1, 2, 3])
+let imm = Immutable.fromList([1, 2, 3])
 ```
 
-### Including Specific Values
+#### Bringing All Values into Scope
 
-Individual items in a module can be brought into scope by placing the included items in curly braces. Similar to with the `provide { ... }` syntax, we need to prefix types with `type` and modules with `module`
+If we want to bring all of the values from another module into scope in our program, we can use an asterisk.
```

Comment:
I think we might want to avoid the term `values` as `use List.*` brings in everything doesn't it, types, submodules, values?
```suggestion
If we want to bring the entire contents of another module into scope in our program, we can use an asterisk.
```